src/services/services.ts

```diff
@@ -376,7 +376,7 @@ namespace ts {
             super(pos, end);
         }
 
-        get unescapedText(): string {
+        get text(): string {
             return unescapeLeadingUnderscores(this.escapedText);

```

Comment:
`delete this.text` would be the worst you could do. V8 will deoptimize the whole object and your optimization becomes a deoptimization.

If you really need to optimize here (I doubt that), you can use `Object.definePropery` to override the getter with the computed value